lib/tasks/installers/angular.rake

```diff
@@ -1,18 +1,14 @@
+require "webpacker/package_json"
+
 namespace :webpacker do
   namespace :install do
     desc "Install everything needed for Angular"
-    task :angular do
-      config_path = Rails.root.join('config/webpack/shared.js')
-
-      config = begin
-        File.read(config_path)
-      rescue Errno::ENOENT
-        puts 'Webpack config not found. Make sure webpacker:install' \
-        ' is run successfully before installing angular'
-        exit!
-      end
+    task angular: ["webpacker:install:verify"] do
+      webpacker_config = Webpacker::PackageJson.webpacker
+      config_path = Rails.root.join(webpacker_config[:configPath], "shared.js")
+      config = File.read(config_path)
 
-      if config.include?('ts-loader')
+      if config.include?("ts-loader")
```

Comment:
Style: Use '' for keys etc without interpolation. Use "" for sentences, prose, or for interpolation.